convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,257 @@
+            )
+            self.s3_client.upload_file(
+                str(badgerdoc_annotations_path),
+                self.s3_output_bucket,
+                s3_output_annotations_path,
+            )
+
+    def request_jobs_to_create_importjob(self):
+        post_importjob_url = f"{settings.job_service_url}create_job/"
+        post_importjob_body = {
+            "name": f"import_job_{uuid4()}",
+            "type": "ImportJob",
+            "import_source": self.s3_input_annotation.path,
+            "import_format": "Label Studio",
+            "owners": [
+                self.token_data.user_id,
+            ],
+            "files": [],
+            "categories": [],
+        }
+        LOGGER.debug(f"{post_importjob_url=}")
+        LOGGER.debug(f"{post_importjob_body=}")
+
+        try:
+            request_to_post_importjob = requests.post(
+                url=post_importjob_url,
+                headers=self.request_headers,
+                json=post_importjob_body
+            )
+            request_to_post_importjob.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise requests.exceptions.RequestException(
+                f"Failed request to 'jobs' - {e}"
+            ) from e
+
+        else:
+            LOGGER.debug(f"{request_to_post_importjob.json()=}")
+            return request_to_post_importjob.json()["id"]
+
+    def request_annotations_to_post_categories(self):
+        post_categories_url = f"{settings.annotation_service_url}categories/"
+        LOGGER.debug(f"{post_categories_url=}")
+
+        pages_objs = self.badgerdoc_format.badgerdoc_annotation.pages[0].objs
+        categories = {pages_obj.category for pages_obj in pages_objs}
+
+        for category in categories:
+            post_categories_body = {
+                "name": category,
+                "type": "box",
+                "id": category,
+            }
+            LOGGER.debug(f"{post_categories_body=}")
+
+            try:
+                request_to_post_categories = requests.post(
+                    url=post_categories_url,
+                    headers=self.request_headers,
+                    json=post_categories_body
+                )
+                request_to_post_categories.raise_for_status()
+            except requests.exceptions.RequestException as e:
+                if request_to_post_categories.status_code == 400 and request_to_post_categories.json() == {
+                    "detail": "Field constraint error. Category id must be unique."
+                }:
+                    LOGGER.info(
+                        f"Category {category} already exists in annotation. Skipping this request"
+                    )
+                else:
+                    raise requests.exceptions.RequestException(
+                        f"Failed request to 'annotation' - {e}"
+                    ) from e
+
+            else:
+                LOGGER.debug(f"{request_to_post_categories.json()=}")
```

Comment:
No need for else here
More specific logging message
```suggestion
            LOGGER.debug(f"{request_to_post_categories.json()=}")
```

Reply:
agreed